src/replica/storage/simple_kv/test/case-207.act

```diff
@@ -64,77 +64,66 @@ client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r1
 config:{4,-,[]}
 state:{}
 
-# assign primary r3
-client:replica_config:receiver=r3,type=assign_primary,node=r3
-config:{5,r3,[]}
-state:{{r3,pri,5,0}}
+# add primary r1
```

Comment:
I noticed that you update 3 unit tests related with learn process, and slao remove original comments. I am not sure will following questions:
- what the original case test for?
- what the difference between the original and tthe new one? can new case still work for the original case?
Could you please fulfill those answers in your pr descption and add comments about the modified process?